radicle-cli/src/commands/ls.rs

```diff
@@ -48,18 +48,19 @@ pub fn run(_options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     let storage = &profile.storage;
     let mut table = term::Table::default();
 
-    storage.repositories()?.into_iter().for_each(|id| {
-        let Ok(repo) = storage.repository(id) else { return };
-        let Ok((_, head)) = repo.head() else { return };
-        let Ok(proj) = repo.project_of(profile.id()) else { return };
+    for id in storage.repositories()? {
+        let Ok(repo) = storage.repository(id) else { break };
+        let Ok((_, head)) = repo.head() else { break };
+        let Ok(proj) = repo.project() else { break };
+
```

Comment:
`break` has different semantics compared to the previous iteration. The `return` was equivalent to a `continue`

Reply:
Fixed!